src/app/ApplicationTemplate.UWP/Views/Styles/Application/Converters.xaml

```diff
@@ -27,4 +27,20 @@
 
 	<converters:DebugConverter x:Key="debug" />
 
+	<!-- Create  -->
+	<converters:FromPasswordStateToCustomValueConverter x:Key="PasswordStateToIconStyle"
+														DefaultValue="{StaticResource CirclePathControlStyle}"
+														ValidValue="{StaticResource SuccessCheckPathControlStyle}"
+														InvalidValue="{StaticResource ErrorXPathControlStyle}" />
+
+	<converters:FromPasswordStateToCustomValueConverter x:Key="PasswordStateToIconMargin"
+														DefaultValue="9,4,9,4"
+														ValidValue="9,4,9,4"
+														InvalidValue="9,4,9,4" />
+
+	<converters:FromPasswordStateToCustomValueConverter x:Key="PasswordStateToMargin"
```

Comment:
It seems very weird to have converters for such small margin values. Can't we have a layout that doesn't need converters?